app/src/main/java/com/example/wellfed/recipe/RecipeActivity.java

```diff
@@ -75,7 +75,7 @@ protected void onCreate(Bundle savedInstanceState) {
         TextView description = findViewById(R.id.recipe_description_textView);
         ImageView img = findViewById(R.id.recipe_img);
 
-        RecipeDB recipeDB = new RecipeDB();
+        RecipeDB recipeDB = new RecipeDB(getApplicationContext(), false);
```

Comment:
I think we should do it this way,
```java
DBConnection connection = new DBConnection(context);
RecipeDB recipeDB = new RecipeDB(connection);
```

Then in tests we can do,
```java
TestDBConnection connection = new TestDBConnection();
RecipeDB recipeDB = new RecipeDB(connection);
```

Reply:
Where TestDBConnection overrides getUUID()

Reply:
> Where TestDBConnection overrides getUUID()

These changes have been made. Thank you for your recommendation!